webpack/components/extensions/Hosts/BulkActions/BulkErrataWizard/04_Review.js

```diff
@@ -0,0 +1,160 @@
+import React, { useContext, useState } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { FormattedMessage } from 'react-intl';
+import { TreeView, Button, Text, TextContent, TextVariants, Flex, FlexItem, Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import { useWizardContext } from '@patternfly/react-core/next';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { BulkErrataWizardContext } from './BulkErrataWizard';
+
+export const dropdownOptions = [
+  __('via remote execution'),
+  __('via customized remote execution'),
+];
+
+export const BulkErrataReview = () => {
+  const { goToStepById } = useWizardContext();
+  const {
+    finishButtonText,
+    selectedRexOption,
+    setSelectedRexOption,
+    finishButtonLoading,
+    errataBulkSelect: {
+      selectedResults: selectedErrataResults,
+      selectedCount: currentSelectedErrataCount,
+    },
+    hostsBulkSelect: {
+      selectedCount: currentSelectedHostsCount,
+    },
+  } = useContext(BulkErrataWizardContext);
+
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const toggleDropdownOpen = () => setIsDropdownOpen(prev => !prev);
+  const handleSelect = () => {
+    setIsDropdownOpen(false);
+  };
+
+  const dropdownItems = dropdownOptions.map(text => (
+    <DropdownItem key={`option_${text}`} ouiaId={`option_${text}`} onClick={() => setSelectedRexOption(text)}>{text}</DropdownItem>
+  ));
+
+  const treeViewTitle = __('Errata to apply');
+  const treeViewData = [
+    {
+      name: treeViewTitle,
+      id: 'errata-treeview-title',
+      customBadgeContent: currentSelectedErrataCount,
+      children:
+        selectedErrataResults.map(({ id, name }) => ({
+          name,
```

Comment:
Would be nice to include the errata ID here too.
```suggestion
        selectedErrataResults.map(({ id, name }) => ({
          name: `${id}: ${name}`,
```

Reply:
Oops.. I was hoping this would be the errata ID. But I am seeing:
![image](https://github.com/Katello/katello/assets/22042343/637af27f-f441-468b-bb94-426a34e7d489)